Packs/CarbonBlackDefense/Integrations/CarbonBlackEndpointStandardV3/CarbonBlackEndpointStandardV3.yml

```diff
@@ -0,0 +1,3751 @@
+      description: The description of the observed process file.
+      type: String
+    - contextPath: CarbonBlackDefense.EventDetails.Results.process_guid
+      description: The GUID of the observed process.
+      type: String
+    - contextPath: CarbonBlackDefense.EventDetails.Results.process_hash
+      description: The hash of the observed process.
+      type: String
+    - contextPath: CarbonBlackDefense.EventDetails.Results.process_integrity_level
+      description: The integrity level of the observed process.
+      type: String
+    - contextPath: CarbonBlackDefense.EventDetails.Results.process_internal_name
+      description: The internal name of the observed process.
+      type: String
+    - contextPath: CarbonBlackDefense.EventDetails.Results.process_name
+      description: The name of the observed process.
+      type: String
+    - contextPath: CarbonBlackDefense.EventDetails.Results.process_original_filename
+      description: The original filename of the observed process.
+      type: String
+    - contextPath: CarbonBlackDefense.EventDetails.Results.process_pid
+      description: The PID of the observed process.
+      type: Number
+    - contextPath: CarbonBlackDefense.EventDetails.Results.process_privileges
+      description: The privileges associated with the observed process.
+      type: String
+    - contextPath: CarbonBlackDefense.EventDetails.Results.process_product_name
+      description: The product name associated with the observed process.
+      type: String
+    - contextPath: CarbonBlackDefense.EventDetails.Results.process_product_version
+      description: The product version associated with the observed process.
+      type: String
+    - contextPath: CarbonBlackDefense.EventDetails.Results.process_publisher
+      description: The publisher of the observed process.
+      type: String
+    - contextPath: CarbonBlackDefense.EventDetails.Results.process_publisher_state
+      description: The state of the publisher of the observed process.
+      type: String
+    - contextPath: CarbonBlackDefense.EventDetails.Results.process_reputation
+      description: The reputation of the observed process.
+      type: String
+    - contextPath: CarbonBlackDefense.EventDetails.Results.process_service_name
+      description: The service name associated with the observed process.
+      type: String
+    - contextPath: CarbonBlackDefense.EventDetails.Results.process_sha256
+      description: The SHA-256 hash.
+  - name: cbd-find-observation-results
+    arguments:
+    - name: polling
+      defaultValue: false
+      hidden: true
+      description: 'When set to false, the function will not use polling and will return the process status or the result if the process is complete.'
+    - name: job_id
+      required: true
+      description: 'The unique identifier for the process search job. This ID is used to retrieve the results of the process search initiated by Carbon Black Cloud.'
```

Comment:
The unique ID for the process search job. This ID is used to retrieve the results of the process search initiated by Carbon Black Cloud.